ptsprojects/stack.py

```diff
@@ -219,6 +219,8 @@ def current_settings_get(self, key):
             return False
 
     def iut_addr_get_str(self):
+        while not self.iut_bd_addr.data["address"]:
```

Comment:
This can be dangerous (infinite loop). It is not related to Python3 so remove this from the PR, you can propose a solution in a different PR.